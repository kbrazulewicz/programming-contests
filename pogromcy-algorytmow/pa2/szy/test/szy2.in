2
29
1947575
168
3044659
4504335
15303
45921
13840195
169344
32956698
13212984
137773
1042334
3720073
33480771
90398128
259917
2008428
18549
413334
30132032
18725481
1379186
11160251
98879048
54466
91240021
13023888
5105
6240842
231798738
