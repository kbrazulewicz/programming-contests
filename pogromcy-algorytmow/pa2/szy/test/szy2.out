10010100111101011010101100100
